source/Error/ConsoleHandler.ts

```diff
@@ -13,4 +13,4 @@ module Magic.Error {
 			console.log(message.toString())
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
Why does this type of change keep appearing in your commits? Does your editor automatically insert newlines at the end of files?